### PR TITLE
Gecko ready to install from npm git

### PR DESCRIPTION
**Pull request by strelok2012**

### CODE REVIEW

File: app/controller.js [hunk 1]
```diff
@@ -109,6 +114,10 @@ class MainController {
                 return;
             }
 
+            const isMacMeta = window.navigator.platform === 'MacIntel' && e.metaKey
```
Comment (GolanLevy) [inline app/controller.js]: I guess this is the right handling for "Ctrl" for mac users. If so, can you implement it on every "Ctrl" used here (for example, Ctrl+leftArrow).

File: app/controller.js [hunk 2]
```diff
@@ -128,6 +137,11 @@ class MainController {
                 self.undo();
             } else if (e.key === 'Enter') {
                 self.playRegion();
+            } else if (isDownCtrl) {
```
Comment (GolanLevy) [inline app/controller.js]: Why does this shortcut here? Isn't Ctrl+leftarrow enough?

File: app/controller.js [hunk 3]
```diff
@@ -1838,6 +1853,19 @@ class MainController {
...
         } else if (keys === 'alt_space') {
             this.playRegion()
+        } else if (which) {
+            if (which === 219) {
+                this.moveBack()
+            }
+        }
+    }
+
+    moveBack () {
```
Comment (GolanLevy) [inline app/controller.js]: Why not use wavesurfer.skip?

Comment (GolanLevy) [inline app/controller.js]: Why not use wavesurfer.skip?

File: app/controller.js (hunk 2 above)
Comment (GolanLevy) [inline app/controller.js]: Why does this shortcut here? Isn't Ctrl+leftarrow enough?

File: app/controller.js (hunk 1 above)
Comment (GolanLevy) [inline app/controller.js]: I guess this is the right handling for "Ctrl" for mac users. 
If so, can you implement it on every "Ctrl" used here (for example, Ctrl+leftArrow).

**Comment by GolanLevy** [comment]
Please make sure that you can run the new version as a github page (like https://gong-io.github.io/gecko/)

you can set a github page on your github account to check that

File: app/controller.js (hunk 3 above)
Comment (strelok2012) [reply app/controller.js]: I've changed my code to use .skip, thanks

File: app/controller.js (hunk 2 above)
Comment (strelok2012) [reply app/controller.js]: It's needed here because Omri said to add back 5 secs shortcut for all modes

File: app/controller.js (hunk 1 above)
Comment (strelok2012) [reply app/controller.js]: changed it

**Comment by strelok2012** [comment]
> Please make sure that you can run the new version as a github page (like https://gong-io.github.io/gecko/)
> 
> you can set a github page on your github account to check that

Done with it

But, now we should use this method https://gist.github.com/cobyism/4730490 to publish build folder as subtree to another branch (i.e. gh-pages) and it works - https://strelok2012.github.io/gecko

**Pull request merged**